### PR TITLE
Bio-Formats 5.7.2 release

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/9Y0PGqsf/28-open-pr-against-wwwopenmicroscopyorg and https://github.com/openmicroscopy/bioformats/pull/3002

N.B. 5.7.2 links (docs & downloads) won't work until the release build is run

### CODE REVIEW

**Comment by hflynn** [comment]
See https://snoopycrimecop.github.io/www.openmicroscopy.org/2017/11/21/bio-formats-5-7-2.html and https://snoopycrimecop.github.io/www.openmicroscopy.org/bio-formats/downloads/ for staging

File: _posts/2017-11-21-bio-formats-5-7-2.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Today we are releasing Bio-Formats 5.7.2 which includes the following changes:
+
+File format fixes and improvements:
+
+* Nikon ND2
+    * fixed a bug which would use the incorrect channel count for small-sized
+      single channel images
+
+* MetaMorph TIFF
+    * changed the reader's behaviour to populate exposure times for all planes
```
Comment (mtbc) [inline _posts/2017-11-21-bio-formats-5-7-2.md]: behavior

File: _posts/2017-11-21-bio-formats-5-7-2.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* added clarification regarding Bio-Formats version requirements for using
+  Java 7 or above
+* updated download links to latest Bio-Formats release version
+* updated the link to the most active fork of JAI ImageIO
+* fixed a number of external broken links
+* added a Trello link for contributing external developers
+* added a link to [Adding format/reader documentation pages](https://docs.openmicroscopy.org/latest/bio-formats/developers/format-documentation.html)
+  to help those contributing to the documentation or supported formats pages
+* the [Bio-Rad Gel page](https://docs.openmicroscopy.org/latest/bio-formats/formats/bio-rad-gel.html) has been updated to add a link to biorad1sc_reader, an
+  external python implementation (thanks to Matthew Clapp)
```
Comment (mtbc) [inline _posts/2017-11-21-bio-formats-5-7-2.md]: maybe initial capital for Python, especially as Java got one above

File: _posts/2017-11-21-bio-formats-5-7-2.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* added clarification regarding Bio-Formats version requirements for using
+  Java 7 or above
+* updated download links to latest Bio-Formats release version
+* updated the link to the most active fork of JAI ImageIO
+* fixed a number of external broken links
+* added a Trello link for contributing external developers
+* added a link to [Adding format/reader documentation pages](https://docs.openmicroscopy.org/latest/bio-formats/developers/format-documentation.html)
+  to help those contributing to the documentation or supported formats pages
+* the [Bio-Rad Gel page](https://docs.openmicroscopy.org/latest/bio-formats/formats/bio-rad-gel.html) has been updated to add a link to biorad1sc_reader, an
+  external Python implementation (thanks to Matthew Clapp)
```
Comment (jburel) [inline _posts/2017-11-21-bio-formats-5-7-2.md]: we will probably need to add an entry to https://www.openmicroscopy.org/contributors/

**Comment by jburel** [comment]
Looks good otherwise

File: _posts/2017-11-21-bio-formats-5-7-2.md (hunk 3 above)
Comment (hflynn) [reply _posts/2017-11-21-bio-formats-5-7-2.md]: I've done that on this PR already

Comment (jburel) [reply _posts/2017-11-21-bio-formats-5-7-2.md]: sorry the changes were hidden being another window
Great thanks

**Comment by hflynn** [comment]
Merging ready for release this pm

**Pull request merged**